src/utils/dom.ts

```diff
@@ -343,3 +343,37 @@ export function slotChangeGetAssignedElements(event: Event): Element[] {
 export function isPrimaryPointerButton(event: PointerEvent): boolean {
   return !!(event.isPrimary && event.button === 0);
 }
+
+/**
+Focus helpers for keyboard navigation
+ *
+ * @param elements : An array of elements
+ * @param elCurrent : The current element
+ * @param destination : "next" | "prev" | "first" | "last"
+ */
+
+export const focusElementInGroup = (elements: Element[], elCurrent: Element, destination: string): void => {
```

Comment:
An enhancement would be to return the focusTarget instead of just void. That would let any component know if something were focused.

Reply:
Like this?

```
export const focusElementInGroup = (elements: Element[], elCurrent: Element, destination: string): Element => {
 
  ...

  focusElement(focusTarget);
  return focusTarget;
};
```

Reply:
Yep!

Reply:
Cool, updated.